test/unit/logger.test.js

```diff
@@ -22,25 +22,25 @@ describe('Logger', () => {
     it('should have blue color when the level is INFO', () => {
       logger.info('test');
       assert.strictEqual(stream.toString(),
-        `${chalk.blue('[INFO]')} test${EOL}`);
+        `${chalk.blue('[INFO]')} test\n`);
```

Comment:
This and the following changes are necessary due to pino using `\n` on all platforms, see https://github.com/pinojs/pino/issues/326.